.github/workflows/main.yml

```diff
@@ -14,34 +14,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby:
-          - '3.0'
-          - '3.1'
-          - '3.2'
-          - '3.3'
-        rails:
-          - '6.1'
-          - '7.0'
-          - '7.1'
-          - '7.2'
-          - '8.0'
         db:
           - mysql
           - postgresql
           - sqlite
-        exclude:
-          - rails: '7.0'
-            ruby: '3.1'
-          - rails: '7.0'
-            ruby: '3.2'
-          - rails: '7.0'
-            ruby: '3.3'
-          - rails: '7.2'
-            ruby: '3.0'
-          - rails: '8.0'
-            ruby: '3.0'
-          - rails: '8.0'
-            ruby: '3.1'
+        include:
+          - { ruby: '3.0', rails: '6.1' }
+          - { ruby: '3.1', rails: '7.0' }
+          - { ruby: '3.2', rails: '7.1' }
+          - { ruby: '3.3', rails: '7.2' }
+          - { ruby: '3.4', rails: '8.0' }
```

Comment:
Wondering is Actions are smart enough to combine include with db. If not, I'll rework it but I want to reduce the number of jobs approximately this way.

Reply:
I think let's go back to the original matrix as it didn't run across multiple database types. It doesn't take too long to run anyway :)

Reply:
Done.